beam/portable_beam.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+readonly BEAM_VERSION="2.6.0-nightly"
+readonly LOCAL_JAR_NAME='beam-runners-flink_2.11-job-server.jar'
+# TODO: find a way to automatically get latest nightly snapshot
+readonly NIGHTLY_SNAPSHOT_PATH="beam-runners-flink_2.11-job-server/2.6.0-SNAPSHOT/beam-runners-flink_2.11-job-server-2.6.0-20180719.181838-16.jar"
+readonly NIGHTLY_SNAPSHOT_REPO="https://repository.apache.org/content/repositories/snapshots/org/apache/beam"
+readonly RELEASE_SNAPSHOT_REPO="http://repo1.maven.org/maven2/org/apache/beam"
+# TODO: add 2.6.0 release snapshot path when it becomes available
+readonly RELEASE_SNAPSHOT_PATH=""
+readonly SERVICE_INSTALL_DIR='/usr/lib/beam-job-service'
+readonly SERVICE_WORKING_DIR='/var/lib/beam-job-service'
+readonly SERVICE_WORKING_USER='yarn'
+
+readonly DOCKER_FLINK_SUBNET="${DOCKER_FLINK_SUBNET:-192.168.33.0/24}"
+readonly DOCKER_FLINK_GATEWAY="${DOCKER_FLINK_GATEWAY:-192.168.33.1}"
+readonly DOCKER_FLINK_DEVICE="${DOCKER_FLINK_DEVICE:-docker-flink0}"
+readonly DOCKER_FLINK_NETWORK_NAME="${DOCKER_FLINK_NETWORK_NAME:-flink}"
+
+function is_master() {
+  local role="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
+  if [[ "$role" == 'Master' ]] ; then
+    true
+  else
+    false
+  fi
+}
+
+function snapshot_url() {
+  if [[ "${BEAM_VERSION}" == '2.6.0-nightly' ]] ; then
```

Comment:
Since we're hard-coding the beam version here, I would just drop this method. We should only keep this if we allow it to be toggled by metadata. I'll leave it up to you as to whether that's worthwhile.

Note that if we decide _not_ to have this (as well as the snapshot repo) be toggled by metadata, we should clean up the file and remove unused references/variables.

Reply:
removed and cleaned.